packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -239,20 +250,35 @@ export class ArtifactCreator {
     // variables are captured in its closure scope. TypeScript compiler
     // currently has no mechanism to detect this and will happily tell you
     // your code is correct when it isn't.
-    await page.evaluate(async () => {
-      const modelBecomesReady = (self as any).modelLoaded ?
```

Comment:
When console.log self object in puppeteer, it's actually the window object and doesn't have modelLoaded property at all(and i think i shouldn't have).  i removed it and run update-screenshots and test for all scenarios (except for dspbr-pt), no errors found. So i think we should just remove it.